t/pipe.t

```diff
@@ -934,8 +934,11 @@ shutdown: merged to stdout
 
 
 
-=== TEST 26: interrupt signals should not break ngx.pipe IO
+=== TEST 26: interrupt signals which break io.popen should not break ngx.pipe IO
```

Comment:
Nothing will produce `EINTR` since we don't use `io.popen` in this test. So I am still confused as to what behavior this test is supposed to assert? Can you clarify?

Reply:
@thibaultcha 
This test case shows that `ngx.pipe` is a little safer replacement to the `io.popen`, because it doesn't suffer from that problem. At that time, there is not `lua_sa_restart` directive.

Reply:
Ok... why not. Marking this as resolved.